classes/userGroup/Repository.php

```diff
@@ -171,7 +171,7 @@ public function getArrayIdByRoleId(int $roleId, ?int $contextId = null): array
      *
      * @return LazyCollection<int, UserGroup>
      */
-    public function getByRoleIds(array $roleIds, int $contextId, ?bool $default = null): LazyCollection
+    public function getByRoleIds(array $roleIds, ?int $contextId, ?bool $default = null): LazyCollection
```

Comment:
Is this change intentional/necessary?

Reply:
Truthfully I don't actually remember . I will test and confirm .